docker/Dockerfile.be

```diff
@@ -9,7 +9,7 @@ FROM adoptopenjdk/openjdk11:jre-11.0.11_9-alpine
 
 COPY --from=build /target/ghs-application-*.jar /server.jar
 
-RUN apk update && apk add git curl
+RUN apk update && apk add git curl && apk add cloc bash
```

Comment:
No need for an additional `apk add` call, just add all your dependencies as part of the previous command:
```bash
RUN apk update && apk add git curl cloc bash
```